diff --git a/node_modules/jsonresume-theme-actual/assets/helper.js b/node_modules/jsonresume-theme-actual/assets/helper.js
index 0157b76..022e15d 100644
--- a/node_modules/jsonresume-theme-actual/assets/helper.js
+++ b/node_modules/jsonresume-theme-actual/assets/helper.js
@@ -1,7 +1,7 @@
 const md = require('markdown-it')()
 const iso = require('iso-3166-1')
 const moment = require('moment')
-
+const differenceInMonths = require("date-fns/differenceInMonths")
 const mdToHtml = (string) => string ? md.render(string) : ''
 
 const calcLocation = (location) => {
@@ -46,10 +46,53 @@ const calcDateRange = (start, end) => {
   } else {
     return null
   }
+  const duration = displayDuration(
+    startDate,
+    endDate || new Date().toISOString()
+  )
+  array.push(duration)
 
   return array.join(' - ')
 }
 
+const calcYearRange = (start, end) => {
+  const startDate = moment(start, 'YYYY-MM-DD').format('YYYY')
+  const endDate = moment(end, 'YYYY-MM-DD').format('YYYY')
+  if (startDate && endDate) {
+    if (startDate === endDate) {
+      return startDate
+    }
+    return `${startDate} - ${endDate}`
+  } else if (startDate) {
+    return `${startDate} - Present`
+  } else if (endDate) {
+    return endDate
+  } else {
+    return null
+  }
+}
+
+const displayDuration = (startDate, endDate) => {
+  const start = new Date(startDate);
+  const end = new Date(endDate);
+  const months = differenceInMonths(end, start);
+  const years = Math.floor(months / 12);
+  const remainingMonths = months % 12;
+  // return `${years} yrs, ${remainingMonths} months`;
+  // pluralize only when necessary
+  let result = "";
+  if (years > 0) {
+    result += `${years} ${years === 1 ? "yr" : "yrs"}`;
+  }
+  if (remainingMonths > 0) {
+    result += `${years > 0 ? ", " : ""}${remainingMonths} ${
+      remainingMonths === 1 ? "mo" : "mos"
+    }`;
+  }
+  return result;
+};
+
+
 const beautifyDate = (date) => moment(date, 'YYYY-MM-DD').format('MMM YYYY')
 
 const beautifyArray = (separator, array) => array.filter(x => x).join(separator)
@@ -79,14 +122,30 @@ const beautifyLink = (string) => {
 
 const validArray = (array) => array !== undefined && array.length > 0
 
+const getIcon = (network) => {
+  switch (network.toLowerCase()) {
+    case 'github':
+      return '💻';
+    case 'linkedin':
+      return '💼';
+    default:
+      return '🌐';
+  }
+}
+
+const beautifyProfileLink = (url) => beautifyLink(url)
+
 module.exports = {
   mdToHtml,
   calcLocation,
   calcDate,
   calcDateRange,
+  calcYearRange,
   beautifyDate,
   beautifyArray,
   arrayToPhrase,
   beautifyLink,
+  beautifyProfileLink,
+  getIcon,
   validArray
 }
diff --git a/node_modules/jsonresume-theme-actual/assets/pug/education.pug b/node_modules/jsonresume-theme-actual/assets/pug/education.pug
index 0766ce5..52b7e7d 100644
--- a/node_modules/jsonresume-theme-actual/assets/pug/education.pug
+++ b/node_modules/jsonresume-theme-actual/assets/pug/education.pug
@@ -4,6 +4,10 @@ if helper.validArray(resume.education)
     div(class='container')
       each edu in resume.education
         div(class='article margin-container-2 no-break')
-          div(class='title markdown')!=helper.mdToHtml(helper.beautifyArray(', ', [edu.studyType, edu.area]))
-          span(class='institution margin-text-2')
-            a(href=edu.url)=edu.institution
+          div(class='header no-alone-1')
+            div(class='table')
+              span(class='title')=edu.studyType
+              span(class='margin-text-1')=edu.area
+              span(class='institution margin-text-1')=edu.institution
+              if edu.startDate
+                span(class='date')=helper.calcYearRange(edu.startDate, edu.endDate)
diff --git a/node_modules/jsonresume-theme-actual/assets/pug/links.pug b/node_modules/jsonresume-theme-actual/assets/pug/links.pug
index dc09c9e..8439426 100644
--- a/node_modules/jsonresume-theme-actual/assets/pug/links.pug
+++ b/node_modules/jsonresume-theme-actual/assets/pug/links.pug
@@ -5,7 +5,9 @@ if helper.validArray(resume.basics.profiles) || resume.basics.url
       ul
         if resume.basics.url
           li(class='article')
-            a(href=resume.basics.url)!=helper.beautifyLink(resume.basics.url)
+            span= 'Personal Website: '
+            a(href=resume.basics.url)!=helper.beautifyProfileLink(resume.basics.url)
         each profile in resume.basics.profiles
           li(class='article margin-text-4')
-            a(href=profile.url)!=helper.beautifyLink(profile.url)
+            span= profile.network + ': '
+            a(href=profile.url)!=helper.beautifyProfileLink(profile.url)
diff --git a/node_modules/jsonresume-theme-actual/assets/styles.css b/node_modules/jsonresume-theme-actual/assets/styles.css
index b75e4f1..29cb68d 100644
--- a/node_modules/jsonresume-theme-actual/assets/styles.css
+++ b/node_modules/jsonresume-theme-actual/assets/styles.css
@@ -72,7 +72,7 @@ li:first-child {
   display: table-cell;
   width: 30%;
   padding-left: 50px;
-  text-align: right;
+  text-align: left;
 }
 
 .summary {
@@ -89,7 +89,7 @@ li:first-child {
 }
 
 .margin-container-2:not(:first-child) {
-  margin-top: 20px;
+  margin-top: 30px;
 }
 
 .margin-container-3:not(:first-child) {
@@ -220,24 +220,24 @@ li:first-child {
   display: block;
   padding: 0 10px;
   margin-bottom: 0;
-  columns: 2;
-  column-width: 310px;
-  column-gap: 10px;
 }
 .main .highlights li {
   font-size: 14px;
   list-style: none;
+  padding-left: 1.2em;
+  text-indent: -1.2em;
 }
 .main .highlights li:before {
   content: "· ";
   vertical-align: middle;
+  padding-right: 0.5em;
 }
 .main .left {
   display: table-cell;
 }
 .main .left .title {
   font-size: 18px;
-  font-weight: bold;
+  font-weight: 600;
   display: block;
 }
 .main .left .subtitle {
@@ -252,6 +252,8 @@ li:first-child {
 .main .right .date {
   font-size: 15px;
   font-weight: lighter;
+  font-style: italic;
+  color: #333;
   white-space: nowrap;
 }
 
@@ -291,13 +293,17 @@ li:first-child {
 .links ul li {
   font-size: 14px;
 }
+.links ul li span,
+.links ul li a {
+  display: block;
+}
 
 .education .title {
-  font-size: 15px;
+  font-size: 18px;
+  font-weight: lighter;
+  padding-bottom: 2px;
 }
 .education .institution {
-  font-size: 15px;
-  font-weight: lighter;
 }
 
 .awards .header .title {
@@ -324,13 +330,12 @@ li:first-child {
 }
 .skills .container {
   display: block;
-  columns: 2;
-  column-gap: 10px;
 }
 .skills .container .article {
   break-inside: avoid-column;
   min-width: 120px;
-  padding-top: 20px;
+  padding-top: 15px;
+  padding-bottom: 5px;
 }
 .skills .container .article span {
   display: block;
diff --git a/node_modules/jsonresume-theme-actual/assets/template.pug b/node_modules/jsonresume-theme-actual/assets/template.pug
index 50433ff..49e0013 100644
--- a/node_modules/jsonresume-theme-actual/assets/template.pug
+++ b/node_modules/jsonresume-theme-actual/assets/template.pug
@@ -15,8 +15,8 @@ html(lang="en")
           include pug/publications
           include pug/volunteer
         div(class='side')
-          include pug/skills
           include pug/education
+          include pug/skills
           include pug/certificates
           include pug/awards
           include pug/references
